docs/pages/blog/material-ui-is-now-mui.md

```diff
@@ -0,0 +1,108 @@
+- Material-UI: the set of foundational MIT React components is now called [**MUI Core**](https://github.com/mui-org/material-ui).
+- Material-UI X: the set of advanced React components is now called [**MUI X**](https://github.com/mui-org/material-ui-x).
+
+The new identity matches with the increased scope of our focus.
+
+## The origin
+
+Material-UI started as a React implementation of the Material Design guidelines in 2014.
+The goal was simple, to empower the React developers to use Material Design.
+The community [loved it](https://news.ycombinator.com/item?id=8582439) and was craving for more, so we kept building.
+
+We have made it smooth for the community to contribute.
+We started actively reviewing pull requests and guiding issues.
+Things started to really take off with [the release of v1](/blog/material-ui-v1-is-out/) in 2018.
+The frontend community was actively migrating to React and in desperate need to move fast.
+
+Before we realized it, we had grown into the most popular React UI components library.
+We were heavily used and customized to build: customer-facing applications, internal tools, and mobile-hybrid applications.
+
+## A larger focus
+
+In our last survey, the number of developers that complained about improving Material Design implementation was down by [60%](/blog/2020-developer-survey-results/#comparison-with-last-year) compared to the year before.
+At the same time, 400% more developers were struggling to customize the components.
+
+It is based on this context that we started to work on v5 in 2019.
+Our primary focus was to revamp the **customization DX**.
+It had become clear that design and DX were key to unlock the next stage of growth.
+
+The new branding comes to support the v5.0.0 release while also creating space for new initiatives that broaden the company horizon. It's a big deal!
+
+Our ultimate goal is to become the most effective and efficient tool to build UIs while making it accessible to the many.
+
+## Our new brand
+
+Let's dig in.
+
+### A new name
+
+We are breaking the strong association with Material Design.
+We have seen too many people confusing Material-UI with a Google effort or a synonym of Material Design.
+
+We are now called **MUI**. It stands for **M**aterials to build **UI**s
+and is pronounced [/ɛmjuːaɪ/](http://ipa-reader.xyz/?text=%C9%9Bmju%CB%90a%C9%AA).
+
+It's shorter, it distinguishes ourselves, it's already how many people abbreviate us.
+More importantly, it allows for the release of products not directly coupled to Material Design, such as unstyled/headless version of the components, a brand new second design system as an alternative to MD, and more.
+
+### A new domain name
+
+A new name wouldn't be so official without a new domain.
+So, we have moved home from https://material-ui.com/ to https://mui.com/, a very easy-to-type (and find) 3 letters domain.
+
+### New logos
+
+We've tightened up the company's logo to match the new branding.
```

Comment:
```suggestion
We've tightened up the company logo to match the new brand.
```

Reply:
"branding" (like copy writing") is a verb. The nouns are "brand", and "copy" respectively.